scripts/create-latest

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+import os
+from subprocess import run, DEVNULL, PIPE
+
+# We *probably* won't change any of these, but at least it means we
+# could if we wanted to.
+LATEST = "latest"
+ORIGIN = "origin"
+MASTER = "master"
+UPDATE = "./src/update_versions_to_latest_master.py"
+
+
+def subprocess_output(x):
+    return x.stdout.decode("utf-8").rstrip().split("\n")
+
+
+def submodule_status():
+    res = run(["git", "submodule", "status"], stdout=PIPE, check=True)
+    lines = subprocess_output(res)
+    parsed = [[x[0] != " "] + x[1:].strip().split(" ")[:2] for x in lines]
+    versions = {x[2]: x[1] for x in parsed}
+    changed = [x[2] for x in parsed if x[0]]
+    return [versions, changed]
+
+
+def get_remote_status():
+    ref = "{}/{}".format(ORIGIN, LATEST)
+    res = run(["git", "rev-parse", ref, "--"], stdout=PIPE, stderr=DEVNULL)
```

Comment:
The `--` is necessary for git to disambiguate references from paths. Because the reference is on the lhs of the `--` git knows its a reference even when it does not exist